packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantRating/RestaurantRating.vue

```diff
@@ -117,6 +117,13 @@ export default {
             type: Number,
             default: 0
         },
+        /**
+         * Max amount of ratings to be shown
+         */
+        countThreshold: {
```

Comment:
Do you think something like `maxDisplayedRatings` might be more descriptive?

Reply:
That is also a good name. But I think this works as well. Also the comment above further more description on its purpose

Reply:
Personally I didn't find the name very clear, and I had to read the comment to understand what it was doing. That's okay when you're in the file but someone using the restaurant card in future might have to come to this file to find out what it does.

I won't block this PR, but I would prefer more descriptive naming.

Reply:
Alright thank you. I will make the change

Reply:
Thanks for making the change. I think you'll also need to update it in the tests and the computed property.